VulkanLaunchpad.cpp

```diff
@@ -411,7 +419,8 @@ std::vector<uint32_t> compileShaderSourceToSpirv(const std::string& shaderSource
 			      << "\n         Log[" << glslang_shader_get_info_log(shader) << "]"
 			      << "\n         Debug-Log[" << glslang_shader_get_info_debug_log(shader) << "]" << std::endl;
 
-		VKL_EXIT_WITH_ERROR("glslang_shader_preprocess failed for " + inputFilename);
+		// VKL_EXIT_WITH_ERROR("glslang_shader_preprocess failed for " + inputFilename);
```

Comment:
The comments in these lines are intentional?
Or can they be removed for the merge?

Reply:
Yeah, they can be removed.